server.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const exphbs = require('express-handlebars');
+const bodyParser = require('body-parser');
+const routeArticles = require('./routes/articles');
+const routeProducts = require('./routes/products');
+
+const app = express();
+const PORT = process.env.PORT || 8080;
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use('/articles', routeArticles);
```

Comment:
These two lines should be at the bottom of the file, just above the `app.listen`